daml-lf/spec/daml-lf-1.rst

```diff
@@ -2327,8 +2418,8 @@ This section lists the built-in functions supported by DAML LF 1.
 The functions come with their types and a description of their
 behavior.
 
-Generic equality function
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Generic equality and order functions
```

Comment:
As for maps, given that we want to switch to ordered map, can we define the order of a map based on the order of the result of converting it to a list of key-value pairs (orderered by keys)?

Reply:
That sounds like the best option :)

Reply:
I added rules for maps. I think cleaning up the spec to get rid of the dependency on insertion order in generic maps should happen in the next PR (if CI ever accepts this one).

Reply:
:+1: